tests/atest/transformers/IndentNestedKeywords/source/settings.robot

```diff
@@ -1,5 +1,6 @@
 *** Settings ***
-Suite Setup    Run Keywords    No Operation    No Operation
+Suite Setup
+...         Run Keywords    No Operation    No Operation
```

Comment:
It's not related to this change, but I've added extra multiline to increase the test coverage in this area.